cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,53 @@
-/// <reference types='cypress' />
-/// <reference types='../support' />
+/* eslint-disable */
+/// <reference types="cypress" />
+/// <reference types="../support" />
+
+import SettingsPageObject from '../support/pages/settingsPageObject'
+import HomePageObject from '../support/pages/home.pageObject'
+
+const settingsPage = new SettingsPageObject()
+const homePage = new HomePageObject()
 
 describe('Settings page', () => {
-  before(() => {
+  let user
 
-  });
+  before(() => {
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser
+    })
+  })
 
   beforeEach(() => {
-
-  });
+    cy.task('db:clear')
+    cy.visit('/#/register')
+    cy.registerAndLogin(user.email, user.username, user.password)
+    settingsPage.visit()
+  })
 
   it('should provide an ability to update username', () => {
+    settingsPage.typeUsername(user.username)
+    settingsPage.clickUpdateSettingsBtn()
 
-  });
+    homePage.assertHeaderContainUsername(user.username)
+  })
 
   it('should provide an ability to update bio', () => {
-
-  });
+    settingsPage.typeBio(user.bio)
+    settingsPage.clickUpdateSettingsBtn()
+  })
```

Comment:
Consider adding an assertion to verify that the bio was updated successfully after clicking the update button.